test/Microsoft.TestPlatform.AcceptanceTests/DeprecateExtensionsPathWarningTests.cs

```diff
@@ -20,18 +20,6 @@ public class DeprecateExtensionsPathWarningTests : AcceptanceTestBase
     private IList<string> _adapterDependencies;
     private IList<string> _copiedFiles;
 
-    private string BuildConfiguration
```

Comment:
This moved to acceptance test base, and there we pick it up from integration environment.